src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/FileWatching/LspContractTypes.cs

```diff
@@ -26,8 +27,9 @@ internal class FileSystemWatcher
 [DataContract]
 internal class RelativePattern
 {
-    [DataMember(Name = "baseUri")]
-    public required string BaseUri { get; set; }
+    [DataMember(Name = "uri")]
```

Comment:
I'm confused by the change to the DataMember name, only because if I'm reading the spec it does say baseURi; I assume this means I'm missing something (or VS Code isn't following the spec?)

Reply:
accidentally changed it in PR cleanup - fixed!